src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/debian/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -23,7 +23,8 @@
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll"/>
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll"/>
     <ArchitectureSpecificToolFile Include="$(BinDir)crossgen"/>
-
+    <ArchitectureSpecificToolFile Include="$(BinDir)libclrjit.so"/>
```

Comment:
If coreclr.dll is to load the clrjit.dll from sxs folder, then you need a copy with coreclr.dll (similar to mscorlib.ni.dll) for unpublished apps to work, if anyone were to consume this package.